### DISCUSSION TITLE
UGS platform won't start

### DISCUSSION THREAD

**Discussion by TartamudoStan** [winder/Universal-G-Code-Sender#1913] [Help] [answered]
Hello
I have a problem.
My ugs only shows the startup image, where you can see that it is starting the configuration and it is only removed and nothing else happens. It was working perfectly until my laptop lost power and shut down. The next day it didn't want to start. I have windows 7 64 bit, I just reinstalled java and it doesn't work either. I use a grbl board, the one that is black. Laptop is used just to the cnc

I hope you can help me.

@breiler asked me 'logs' but I don't know how to get that. My UGS version is 2.0.7

I can't provide screenshots because literally nothing happens, just the startup image and it disappears

**Comment by breiler** [comment, marked as answer]
You will find the log files here:
C:/Users/[your username>]/.ugsplatform/2.X/var/log/messages.log

**Reply by TartamudoStan** [reply]
Hello
Sorry I hardly had any free time here are my files.

[messages.log](https://github.com/winder/Universal-G-Code-Sender/files/9161574/messages.log)

**Reply by breiler** [reply]
It looks like the configuration has become corrupt. 

You can first try and delete the cache directory under:
```C:\Users\Dany\.ugsplatform\2.0.7-SNAPSHOT\dev\var\cache```

If that doesn't work, try and delete the whole:
```C:\Users\Dany\.ugsplatform\2.0.7-SNAPSHOT```

**Reply by TartamudoStan** [reply]
Thank you so much you saved me
I had to delete the entire 2.0.7-SNAPSHOT folder
But it already started.
Thaks
Regards

**Comment by liamc96-dotcom** [comment]
[messages.log](https://github.com/winder/Universal-G-Code-Sender/files/9128592/messages.log)

**Reply by breiler** [reply]
I don't think so, you are running a UGS version that you built yourself? @TartamudoStan is using a prebuilt version 2.0.7 with a bundled Java.

You are trying to run it with Java 18 which is not supported by the Netbeans version we are using. Read this on how to get started building the software: https://github.com/winder/Universal-G-Code-Sender#development
